### DISCUSSION TITLE
how to change the category?

### DISCUSSION THREAD

**Discussion by ura2rist** [community/community#23333] [New to GitHub]
<p>how to change the category? </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d342d7fe810fe10010dafbf45443a547f8a18dcd.jpeg" data-download-href="/uploads/short-url/u8TXpFh7MG7h5PI5GX8Kngi7tnv.jpg?dl=1" title="UeU4siEGJ7Q.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/53140220/181076181-0a8eeebd-e033-431f-a5e6-7b531c49c991.jpg" alt="UeU4siEGJ7Q.jpg" data-base62-sha1="u8TXpFh7MG7h5PI5GX8Kngi7tnv" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d342d7fe810fe10010dafbf45443a547f8a18dcd_2_690x388.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/d342d7fe810fe10010dafbf45443a547f8a18dcd_2_1035x582.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/d342d7fe810fe10010dafbf45443a547f8a18dcd_2_1380x776.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d342d7fe810fe10010dafbf45443a547f8a18dcd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">UeU4siEGJ7Q.jpg</span><span class="informations">1920×1080 233 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Gisgar3** [comment]
<p>You are seeing  <strong>CSS</strong> as your main language because it is the most-used in that repository. Other than adding more source languages to your repository, I do not think there is any manual way to change that displayed language.</p>
<p>Hope this helps!</p>

**Reply by MUHAMMADTAHIR1122** [reply]
1. Go to your repository.


2. In the "About" section, click the gear icon next to "Topics."


3. Add or update topics that best represent the category.

[
4. Click Save to apply the changes](https://profaceapk.com/).

**Reply by faceapphub112** [reply]
You can’t directly change the category of an existing repository on GitHub. Instead, you can update your repository topics (tags) or update the description to better match the category you want.

Go to your repo → click Settings → under Topics section, add or edit the keywords. This helps GitHub and others understand the project’s category.

 If you’d like a step-by-step explanation, you can [check this guide 👈](https://thegamespike.com/)

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/ura2rist">@ura2rist</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>You cannot change the language of the repository, but you can change the attributes of the github repository. I mean that if you have a project where there are 60% css and 40% javascript you can said to <a href="https://github.com/github/linguist" rel="nofollow noopener">github-linguist</a>, that you want to ignore the css file.</p>
<p>To do that you need to:</p>
<ol>
<li>
<p>create a .gitattributes file in the root of the project</p>
</li>
<li>
<p>edit that file and add a line to ignore the css language file</p>
<p>*.css linguist-detectable=false<br>
*.js linguist-detectable=true</p>
</li>
<li>
<p>add, commit and push to remote and see the result</p>
</li>
</ol>
<p>With this 3 steps you can ignore detection for a specific languages. You can also add more attributes, for example something like this:</p>
<pre><code>*.css linguist-detectable=false
*.js linguist-detectable=true
*.java linguist-detectable=false
</code></pre>
<p>this attributes ignore the java files.</p>
<p>Hope this helps…</p>
<p>Ps: if you need some screenshots ask…</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Reply by pankajAdhikari2002** [reply]
Thanks man helped a lot

**Reply by Ayu85** [reply]
Thanks man , time and life saver you are...

**Reply by itsAbdullahMaqsood** [reply]
Damn, This was simple, concise and to the point. Helped alot, Thanks!

**Reply by raufzer** [reply]
Thank you for your solution mate.

**Comment by qinze2017** [comment]
<p>The solution for ignoring html files could be found here <a href="https://github.com/github/linguist/issues/4769" rel="nofollow noopener">https://github.com/github/linguist/issues/4769</a></p>

**Comment by markr0v** [comment]
just create .gitattributes in the root folder of your repository, and give it these contents:

```
* linguist-vendored
*.dart linguist-vendored=false
```

This example tells GitHub/Linguist to ignore all files and look at .dart files. You can also add another language you want as long as the format files are included in your repository

**Reply by mobin-aura** [reply]
thank you !

**Comment by dbatu00** [comment]
You can find language aliases here:
https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml

**Comment by rameshgyawali127** [comment]
#Response 1
